pkg/agent/baker.go

```diff
@@ -220,7 +220,7 @@ func getContainerServiceFuncMap(config *NodeBootstrappingConfiguration) template
 			if profile.KubernetesConfig == nil {
 				return ""
 			}
-			return getDynamicKubeletConfigFileContent(profile.KubernetesConfig.KubeletConfig)
+			return GetDynamicKubeletConfigFileContent(profile.KubernetesConfig.KubeletConfig, profile.CustomKubeletConfig)
```

Comment:
I would not call it dynamic, either remove it or call it custom

Reply:
This func is no longer needed since content is moved to template.

Reply:
do we still need this?

Reply:
Sorry I thought this is for sysctl. It is still needed for getting kubelet config file content when toggle is on.

Reply:
if you don't want to rename this in this PR let's create a new PR to address that.

Reply:
Yes we should rename it. I'll do it in a separate PR since the change is not trivial, don't want to mess with current changes.